lcls-twincat-optics/lcls_twincat_optics_plc/POUs/FB_RunHOMS.TcPOU

```diff
@@ -0,0 +1,102 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.12">
+  <POU Name="FB_RunHOMS" Id="{ee7498ff-903e-47a0-a649-7587adeefcee}" SpecialFunc="None">
+    <Declaration><![CDATA[FUNCTION_BLOCK FB_RunHOMS
+VAR_INPUT
+	// Encoder Reference Values
+	nYupEncRef : ULINT;
+	nYdwnEncRef : ULINT;
+	nXupEncRef : ULINT;
+	nXdwnEncRef : ULINT;
+
+	// Gantry tolerances
+	nGantryTolY : LINT := 50000; // Encoder counts = nm
```

Comment:
Maybe make `50000` a constant such as `GANTRY_TOLERANCE_NM_DEFAULT`?

Reply:
Should these even be global constants? They come straight off of a spec sheet and may be used in multiple places.

Reply:
That makes sense to me. I added them to `GVL_HOMS` which I originally thought could live in the library itself since this parameter is common to all the Optics. Though now I'm realizing this should really live in specific Optics GVL lists such as `GVL_M1L0` since it needs to get passed to instances of `FB_RunHOMS`, or maybe this shouldn't be an input variable at all?

Reply:
Resolved in commit fbe4fcc